### PR TITLE
Add extra col to represent no. of ops

### PR DESCRIPTION
**Pull request by ManwithaYarradoddi**
case: https://github.ibm.com/cloudant/fdb-acceptance/issues/107

### CODE REVIEW

File: src/basho_bench_stats_writer_csv.erl [hunk 1]
```diff
@@ -129,24 +129,28 @@ report_global_stats({Op,_}, Stats, Errors, Units) ->
...
-               Stats, Errors, Units) ->
+               Stats, Errors, Units, Operations) ->
     case proplists:get_value(n, Stats) > 0 of
         true ->
             P = proplists:get_value(percentile, Stats),
             Line = io_lib:format("~w, ~w, ~w, ~w, ~.1f, ~w, ~w, ~w, ~w, ~w, ~w\n",
                                  [Elapsed,
                                   Window,
                                   Units,
+                                  Operations,
```
Comment (chewbranca) [inline src/basho_bench_stats_writer_csv.erl]: You're adding another parameter to the `io_lib:format` call above but not to the format string. You'll want to add another `~w` to the format string a few lines above.

Comment (ManwithaYarradoddi) [reply src/basho_bench_stats_writer_csv.erl]: i missed it

File: src/basho_bench_stats.erl [hunk 2]
```diff
@@ -72,7 +72,9 @@ op_complete(Op, {ok, Units}, ElapsedUs) ->
             folsom_metrics:notify_existing_metric({latencies, Op}, ElapsedUs, histogram),
             folsom_metrics:notify_existing_metric({overall_latencies, Op}, ElapsedUs, histogram),
             folsom_metrics:notify_existing_metric({overall_units, Op}, {inc, Units}, counter),
-            folsom_metrics:notify_existing_metric({units, Op}, {inc, Units}, counter)
+            folsom_metrics:notify_existing_metric({units, Op}, {inc, Units}, counter),
+            folsom_metrics:notify_existing_metric({overall_opz, Op}, {inc, 1}, counter),
```
Comment (chewbranca) [inline src/basho_bench_stats.erl]: Could you change `opz` to `ops` please? Basically use `s` instead of `z`.

**Review by chewbranca — CHANGES_REQUESTED** [review]

**Review by chewbranca — APPROVED** [review]
Looks good!

**Pull request merged**